content/en/docs/languages/csharp/basics.md

```diff
@@ -1,484 +0,0 @@
----
```

Comment:
For every page that is dropped, like this one, we need to add a redirect rule.

Reply:
done.